src/pages/Dashboard/Activities/index.tsx

```diff
@@ -0,0 +1,103 @@
+import { useState } from 'react';
+// import { sendSignInLinkToEmail } from 'firebase/auth';
+// import axios from 'axios';
+
+// import { auth } from '../../../firebase';
+import Table from '../Activities/ActivityTable';
+import Button from '../../../components/Button';
+import SearchBar from '../../../features/SearchBar';
+import PageHeader from '../../../features/PageHeader';
+import ListBanner from '../../../features/ListBanner';
+import AddForm, { IFormData } from './AddForm';
+import { ITab } from '../../../features/TabList';
+import { IconNames } from '../../../components/Icon';
+import * as constant from './constant';
+
+const tabs: ITab[] = [
+  { label: constant.ALL_ACTIVITY_LABEL },
+];
+
+// TODO: --- THIS IS A PLACEHOLDER --- Replace with real component.
+const Box = (): JSX.Element => (
+  <div className='border rounded-sm w-36 p-1.5 border-gray-300 flex flex-row items-center bg-gray-300 text-white'>
+    <span className="material-symbols-outlined">expand_more</span>
+    <span className='flex-1'>User</span>
+    <span className="material-symbols-outlined">account_box</span>
+  </div>
+);
+
+const ActivityDashboard = (): JSX.Element => {
+  const [isOpen, setIsOpen] = useState(false);
+  const businessId = 1234; // TODO: REPLACE THIS WITH A GLOBAL VARIABLE
```

Comment:
To access the businessId, you can now use the `useParams()` hook from `react-router`.
```
const { businessId } = useParams();
```
it will get the `businessId` from the url